pkgs/development/libraries/librdf/raptor2.nix

```diff
@@ -11,6 +23,11 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-h03IyFH1GHPqajfHBBTb19lCEu+VXzQLGC1wiEGVvgY=";
   };
 
+  cmakeFlags = [
+    # Build defaults to static libraries.
```

Comment:
should we ask upstream to build shared instead of static by default?

Reply:
Sounds reasonable. Asked as https://github.com/dajobe/raptor/pull/53